src/planscape/planning/views.py

```diff
@@ -229,8 +229,8 @@ def get_planning_area_by_id(request: HttpRequest) -> HttpResponse:
 
     Returns: The planning area in JSON form.  The JSON will also include two metadata fields:
       scenario_count: number of scenarios for this planning area.
-      scenario_latest_updated_at: latest datetime (e.g. 2023-09-08T20:33:28.090393Z) across all scenarios or
-                                  None if there are no scenarios.
+      latest_updated: latest datetime (e.g. 2023-09-08T20:33:28.090393Z) across all scenarios or
+        Plan updated_at if no scenarios
```

Comment:
Nit: For consistency can you refer to this as a Planning Area? (Sorry for the small nit but it helps to keep things from getting too confusing with the deprecated "plan" app models)